hbase-client/src/main/java/org/apache/hadoop/hbase/client/Connection.java

```diff
@@ -109,16 +109,15 @@ default Table getTable(TableName tableName, ExecutorService pool) throws IOExcep
   /**
    * <p>
    * Retrieve a {@link BufferedMutator} for performing client-side buffering of writes. The
-   * {@link BufferedMutator} returned by this method is thread-safe. This BufferedMutator will
-   * use the Connection's ExecutorService. This object can be used for long lived operations.
+   * {@link BufferedMutator} returned by this method is thread-safe.
+   * This accessor will create a new ThreadPoolExecutor and will be shutdown once we close the
+   *  BufferedMutator. This object can be used for long lived operations.
```

Comment:
Nit: May be for API getBufferedMutator(BufferedMutatorParams params) we can say clearly that if user passes a ThreadPool in params, we will use that or else same as this method (new one and handled by us on close)

Reply:
@anoopsjohn  Done ! Please review again !